org.scram_pra.scram.yml

```diff
@@ -0,0 +1,42 @@
+app-id: org.scram_pra.scram
+runtime: org.kde.Sdk
+runtime-version: 5.15-21.08
+sdk: org.kde.Sdk
+command: scram-gui
+rename-icon: scram
+rename-desktop-file: scram-gui.desktop
+finish-args:
+- "--socket=wayland"
+- "--socket=fallback-x11"
+- "--filesystem=home"
+- "--share=ipc"
+# - "--device=dri" # TODO: needed?
+modules:
+- boost.json
+# libxml2 is already in the sdk
+- name: scram
+  buildsystem: cmake-ninja
+  config-opts:
+  - -DCMAKE_BUILD_TYPE=Release
+  - -DINSTALL_PREFIX=/app
+  - -DBUILD_GUI=ON
+  - -DWITH_TCMALLOC=OFF
+  - -DWITH_JEMALLOC=OFF
+  - -DWITH_COVERAGE=OFF
+  - -DWITH_PROFILE=OFF
+  builddir: true
+  post-install:
+  - desktop-file-install --dir=/app/share/applications gui/scram-gui.desktop
+  - install -Dm755 gui/scram-gui.appdata.xml /app/share/metainfo/org.scram_pra.scram.appdata.xml
+  - install -Dm755 gui/images/scram_log.svg /app/share/icons/hicolor/scalable/apps/scram.svg
+  - install -Dm755 gui/images/scram16x16.png /app/share/icons/hicolor/16x16/apps/scram.png
+  - install -Dm755 gui/images/scram32x32.png /app/share/icons/hicolor/32x32/apps/scram.png
+  - install -Dm755 gui/images/scram48x48.png /app/share/icons/hicolor/48x48/apps/scram.png
+  - install -Dm755 gui/images/scram64x64.png /app/share/icons/hicolor/64x64/apps/scram.png
+  - install -Dm755 gui/images/scram128x128.png /app/share/icons/hicolor/128x128/apps/scram.png
+  - install -Dm755 gui/images/scram256x256.png /app/share/icons/hicolor/256x256/apps/scram.png
```

Comment:
I also see the appdata and the icons being installed by the build system so this is not needed.

Reply:
done